jobrunner/agent/metrics.py

```diff
@@ -125,18 +121,10 @@ def main():  # pragma: no cover
         time.sleep(max(2, config.STATS_POLL_INTERVAL - elapsed))
 
 
-def record_tick_trace(last_run, active_jobs):
-    """Record a period tick trace of current jobs.
-
-    This will give us more realtime information than the job traces, which only
-    send spans data when *leaving* a state.
-
-    The easiest way to filter these in honeycomb is on tick==true attribute
-
-    Not that this will emit number of active jobs + 1 events every call, so we
-    don't want to call it on too tight a loop.
-    """
+def record_metrics_tick_trace(last_run):
+    """Record a period metrics ick trace of current docker containers"""
```

Comment:
```suggestion
    """Record a period metrics tick trace of current docker containers"""
```

Reply:
Gonna not take your suggestion, or you won't be able re-approve